R/MsBackendDataFrame.R

```diff
@@ -248,14 +248,31 @@ setReplaceMethod("smoothed", "MsBackendMemory", function(object, value) {
 })
 
 #' @rdname hidden_aliases
-setMethod("spectraData", "MsBackendMemory", function(object, columns) {
-    ## TODO @jo IMPLEMENT:
-    ## allow to specify columns. Have then to check if these columns exist,
-    ## if not call the respective method.
+#'
+#' @importFrom methods as
+setMethod("spectraData", "MsBackendDataFrame", function(object, columns) {
+    if (missing(columns))
```

Comment:
Same as above, I don't see why you take the union of actual and mandatory column names.